storm-server/src/main/java/org/apache/storm/daemon/supervisor/Container.java

```diff
@@ -219,11 +201,11 @@ public void kill() throws IOException {
             shutdownTimer = shutdownDuration.time();
         }
         try {
-            Set<Long> pids = getAllPids();
+        Set<Long> pids = getAllPids();
 
-            for (Long pid : pids) {
-                kill(pid);
-            }
+        for (Long pid : pids) {
+            kill(pid);
+        }
```

Comment:
It looks like indentation is off around here.  could be spaces vs tabs or something, not really sure.

Reply:
Will reformat